diff -up kdiamond-19.08.2/src/game-state.cpp.omv~ kdiamond-19.08.2/src/game-state.cpp
--- kdiamond-19.08.2/src/game-state.cpp.omv~	2019-10-14 22:17:16.619888995 +0200
+++ kdiamond-19.08.2/src/game-state.cpp	2019-10-14 22:18:49.361411252 +0200
@@ -19,7 +19,7 @@
 #include "game-state.h"
 #include "settings.h"
 
-#include <QTime>
+#include <QElapsedTimer>
 #include <KLocalizedString>
 #include <KNotification>
 
@@ -32,7 +32,7 @@ public:
     GameStatePrivate();
     ~GameStatePrivate();
 
-    QTime m_gameTime, m_pauseTime;
+    QElapsedTimer m_gameTime, m_pauseTime;
 
     Mode m_mode;
     State m_state;
diff -up kdiamond-19.08.2/src/game-state.h.omv~ kdiamond-19.08.2/src/game-state.h
diff -up kdiamond-19.08.2/src/main.cpp.omv~ kdiamond-19.08.2/src/main.cpp
--- kdiamond-19.08.2/src/main.cpp.omv~	2019-10-14 22:16:54.876763118 +0200
+++ kdiamond-19.08.2/src/main.cpp	2019-10-14 22:17:02.885809647 +0200
@@ -46,7 +46,7 @@ int main(int argc, char **argv)
 
     qsrand(time(nullptr));
     KLocalizedString::setApplicationDomain("kdiamond");
-    KAboutData about(QStringLiteral("kdiamond"), i18nc("The application's name", "KDiamond"), QLatin1Literal(version), i18n(description),
+    KAboutData about(QStringLiteral("kdiamond"), i18nc("The application's name", "KDiamond"), QLatin1String(version), i18n(description),
                      KAboutLicense::GPL, i18n("(C) 2008-2010 Stefan Majewsky and others"), QStringLiteral("http://games.kde.org/kdiamond"));
     about.addAuthor(i18n("Stefan Majewsky"), i18n("Original author and current maintainer"), QStringLiteral("majewsky@gmx.net"));
     about.addAuthor(i18n("Paul Bunbury"), i18n("Gameplay refinement"), QStringLiteral("happysmileman@googlemail.com"));
